frontend-react/src/resources/AuthResource.ts

```diff
@@ -19,6 +20,7 @@ export default class AuthResource extends Resource {
             ...init,
             headers: {
                 ...init.headers,
+                ...getAppInsightsHeaders(),
```

Comment:
I realize we're in the process of migrating from old fetch patterns to React Query, but I figured we should get this in in the meantime.  Happy to remove it, though!

Reply:
Spot on, good lookin' out!